Automattic/sensei
# Update colors and typography of default Learning Mode template

Resolves #6736 (just colors & typography, spacing & alignment will be added to a new issue).

## Proposed Changes
* Updates the colors and typography for class themes for the default Learning Mode template.
* Adds a new `colors.scss` file where we can collect colors that are used in multiple places in LM. I thought about making these CSS variables but I'm not sure if that's better?

## Notes
* The CSS in `mobile.scss` is not mobile first, but that's because they were already using `max-width` instead of `min-width`, and I didn't want to break anything by changing it.
* ~Course title isn't styled because I need the changes in https://github.com/Automattic/sensei/pull/6762.~ Course title is now styled.

## Testing Instructions


1. Enable Divi or Astra.
2. Enable the default Learning Mode template.
3. View a lesson in Learning Mode.
4. Check that the colors and typography match the designs.
5. Check out [this branch](https://github.com/Automattic/themes/pull/7008).
6. Enable the Course theme.
7. View a lesson in Learning Mode.
8. Check that the colors and typography match the designs.

## Pre-Merge Checklist
